### DISCUSSION TITLE
Node sync light

### DISCUSSION THREAD

**Discussion by fsl-kyungmin** [ConsenSys/quorum#1606] [Q&A]
#### System information

Geth version: `Geth/v1.10.3-stable-28721bf9(quorum-v22.7.6)/linux-arm64/go1.18.1`
OS & Version: Ubuntu 22(Linux)
Commit hash : (if `develop`)

#### Expected behaviour
want to use --syncmode light

#### Actual behaviour
Got Fatal Error
WARN [02-17|08:31:20.673] Light client mode is an experimental feature 
Fatal: Error retrieving Ethereum service: unknown servic

#### Steps to reproduce the behaviour
Use this command
Already Deploy on Quorum node as QBFT Algorithm
```geth --datadir /home/ubuntu/store2 --networkid 7777 --syncmode light```

#### Backtrace
INFO [02-17|08:31:20.643] Starting Geth on Ethereum mainnet... 
INFO [02-17|08:31:20.643] Dropping default light client cache      provided=1024 updated=128
INFO [02-17|08:31:20.644] Running with private transaction manager disabled - quorum private transactions will not be supported 
INFO [02-17|08:31:20.646] Maximum peer count                       ETH=0 LES=10 total=50
INFO [02-17|08:31:20.646] Smartcard socket not found, disabling    err="stat /run/pcscd/pcscd.comm: no such file or directory"
INFO [02-17|08:31:20.647] Enabling recording of key preimages since archive mode is used 
INFO [02-17|08:31:20.647] Set global gas cap                       cap=25,000,000
INFO [02-17|08:31:20.647] Running with private transaction manager disabled - quorum private transactions will not be supported 
INFO [02-17|08:31:20.647] Allocated cache and file handles         database=/root/.ethereum/geth/lightchaindata cache=64.00MiB handles=524,288
INFO [02-17|08:31:20.655] Allocated cache and file handles         database=/root/.ethereum/geth/les.client     cache=16.00MiB handles=16
INFO [02-17|08:31:20.662] Initialised chain configuration          config="{ChainID: 1 Homestead: 1150000 DAO: 1920000 DAOSupport: true EIP150: 2463000 EIP155: 2675000 EIP158: 2675000 Byzantium: 4370000 IsQuorum: false Constantinople: 7280000 TransactionSizeLimit: 0 MaxCodeSize: 0 Petersburg: 7280000 Istanbul: 9069000, Muir Glacier: 9200000, Berlin: 12244000  Catalyst: <nil> YOLO v3: <nil> PrivacyEnhancements: <nil> PrivacyPrecompile: <nil> EnableGasPriceBlock: <nil> Engine: ethash}"
INFO [02-17|08:31:20.662] Added trusted checkpoint                 block=12,189,695 hash=50fd3c..51e9c9
INFO [02-17|08:31:20.662] Loaded most recent local header          number=0 hash=d4e567..cb8fa3 td=17,179,869,184 age=53y10mo3w
INFO [02-17|08:31:20.663] Configured checkpoint oracle             address=0x9a9070028361F7AAbeB3f2F2Dc07F82C4a98A02a signers=5 threshold=2
WARN [02-17|08:31:20.663] Unclean shutdown detected                booted=2023-02-17T08:14:43+0000 age=16m37s
WARN [02-17|08:31:20.663] Unclean shutdown detected                booted=2023-02-17T08:15:01+0000 age=16m19s
WARN [02-17|08:31:20.663] permission-config.json file is missing. Smart-contract-based permission service will be disabled error="stat /root/.ethereum/permission-config.json: no such file or directory"
INFO [02-17|08:31:20.663] No plugins to initialise 
INFO [02-17|08:31:20.663] Starting peer-to-peer node               instance=Geth/v1.10.3-stable-28721bf9(quorum-v22.7.6)/linux-arm64/go1.18.1
INFO [02-17|08:31:20.671] New local node record                    seq=5 id=0480bbe61ad2d874 ip=127.0.0.1 udp=30303 tcp=30303
INFO [02-17|08:31:20.672] Started P2P networking                   self=enode://09ddc84893dd7581df0a618f16cecd0ba3b3e22b00bbf789b7be3668fa8f6ab22d96c0bc37f68f7745d5f29e39340a69ec58b171f8a90ee689c9cdc6829ea22e@127.0.0.1:30303
INFO [02-17|08:31:20.673] IPC endpoint opened                      url=/root/.ethereum/geth.ipc isMultitenant=false
INFO [02-17|08:31:20.673] Security Plugin is not enabled 
WARN [02-17|08:31:20.673] Light client mode is an experimental feature 
Fatal: Error retrieving Ethereum service: unknown service

````
[backtrace]
````

When submitting logs: please submit them as text and not screenshots.

**Comment by baptiste-b-pegasys** [comment]
sync mode has to be full, mainly because of the private state

**Reply by frank-lim-partior** [reply]
What if a new node only wishes to sync the public state? Perhaps from a node that have not received any private transactions

**Comment by fsl-kyungmin** [comment]
understood. as far as know Quorum support the couple more modes(snap, light, fast)
 it is not working, if working that modes could you share the guide kindly??

**Comment by fsl-kyungmin** [comment]
can use full,snap,fast

Only cannot use light.. 
Version : Geth/v1.10.3-stable-28721bf9(quorum-v22.7.6)/linux-arm64/go1.18.1
<img width="1115" alt="스크린샷 2023-02-28 오전 9 54 14" src="https://user-images.githubusercontent.com/121072152/221723962-3972096b-425f-4ce0-8ade-f15654c1a238.png
<img width="924" alt="스크린샷 2023-02-28 오전 9 55 28" src="https://user-images.githubusercontent.com/121072152/221724135-e3da5baf-4e1a-47a7-8717-e48b4f6721c0.png">
">
